concept/type/owns.feature

```diff
@@ -0,0 +1,3952 @@
+    When entity(person) set owns: custom-attribute
+    Then entity(person) get owns(custom-attribute) set annotation: @card; fails
+    Then entity(person) get owns(custom-attribute) set annotation: @card(); fails
+    Then entity(person) get owns(custom-attribute) set annotation: @card(1); fails
+    Then entity(person) get owns(custom-attribute) set annotation: @card(*); fails
+    Then entity(person) get owns(custom-attribute) set annotation: @card(-1, 1); fails
+    Then entity(person) get owns(custom-attribute) set annotation: @card(0, 0.1); fails
+    Then entity(person) get owns(custom-attribute) set annotation: @card(0, 1.5); fails
+    Then entity(person) get owns(custom-attribute) set annotation: @card(*, *); fails
+    Then entity(person) get owns(custom-attribute) set annotation: @card(0, **); fails
+    Then entity(person) get owns(custom-attribute) set annotation: @card(1, 2, 3); fails
+    Then entity(person) get owns(custom-attribute) set annotation: @card(1, "2"); fails
+    Then entity(person) get owns(custom-attribute) set annotation: @card("1", 2); fails
+    Then entity(person) get owns(custom-attribute) set annotation: @card(2, 1); fails
+    Then entity(person) get owns(custom-attribute) get annotations is empty
+    When transaction commits
+    When connection open read transaction for database: typedb
+    Then entity(person) get owns(custom-attribute) get annotations is empty
+    Examples:
+      | value-type |
+      | long       |
+      | double     |
+      | decimal    |
+      | string     |
+      | boolean    |
+      | datetime   |
+      | datetimetz |
+      | duration   |
+
+  Scenario Outline: Owns cannot set multiple @card annotations with different arguments
+    When put attribute type: custom-attribute
+    When attribute(custom-attribute) set value-type: decimal
+    When entity(person) set owns: custom-attribute
+    When entity(person) get owns(custom-attribute) set annotation: @card(2, 5)
+    Then entity(person) get owns(custom-attribute) set annotation: @card(<fail-args>); fails
+    Then entity(person) get owns(custom-attribute) get annotations contain: @card(2, 5)
+    Then entity(person) get owns(custom-attribute) get annotations do not contain: @card(<fail-args>)
+    When transaction commits
+    When connection open read transaction for database: typedb
+    Then entity(person) get owns(custom-attribute) get annotations contain: @card(2, 5)
+    Then entity(person) get owns(custom-attribute) get annotations do not contain: @card(<fail-args>)
+    Examples:
+      | fail-args |
+      | 0, 1      |
+      | 0, 2      |
+      | 0, 3      |
+      | 0, 5      |
+      | 0, *      |
+      | 2, 3      |
+      | 2, 5      |
+      | 2, *      |
+      | 3, 4      |
+      | 3, 5      |
+      | 3, *      |
+      | 5, *      |
+      | 6, *      |
+
+    # TODO: Maybe we allow it, then change the test considering the expected behavior
```

Comment:
?

Reply:
Yes!

Reply:
I guess we agreed that we don't want to, so I just remove the TODO

Reply:
We agreed that we want `set annotation` to act like `set owns` and `set plays`: `set owns` redeclares it without annotations etc, so `set annotation` should just redeclare its args without errors, I believe.

I'll rewrite these tests to expect args overrides for now.